bot.py

```diff
@@ -17,16 +17,24 @@
 	intents.guilds = True
 	intents.members = True
 
-	client = commands.Bot("~", intents=intents)  # bot command prefix
+	global PREFIX
+	PREFIX = "++"
+	client = commands.Bot(PREFIX, intents=intents)  # bot command prefix
 
-	# Get the base file names of all the files in the modules folder
-	folders = filter(
-		lambda folder: os.path.exists("modules/" + folder + "/cog.py"),
-		os.listdir("modules"),
-	)
+	# Get the modules of all cogs whose directory structure is modules/<module_name>/cog.py
+	for folder in os.listdir("modules"):
+		if os.path.exists(os.path.join("modules", folder, "cog.py")):
+			client.load_extension("modules." + folder + ".cog")
 
-	for folder in folders:
-		client.load_extension("modules." + folder + ".cog")
+	# TODO: remove this code by February (or whenever people get used to the new prefix)
+	@client.event
+	async def on_message(message: discord.Message):
+		if message.content.startswith("~"):
```

Comment:
Technically this could get triggered when not trying to use the bot. For example using as an approx symbol.